--- a/defaults/preferences/zotero.js
+++ b/defaults/preferences/zotero.js
@@ -160,7 +160,7 @@
 // Connector settings
 pref("extensions.zotero.httpServer.enabled", true);
 pref("extensions.zotero.httpServer.port", 23119);	// ascii "ZO"
-pref("extensions.zotero.httpServer.localAPI.enabled", false);
+pref("extensions.zotero.httpServer.localAPI.enabled", true);
 
 // Zeroconf
 pref("extensions.zotero.zeroconf.server.enabled", false);
